pkg/provider/azure.go

```diff
@@ -742,6 +745,10 @@ func (az *Cloud) getPutVMSSVMBatchSize() int {
 }
 
 func (az *Cloud) initCaches() (err error) {
```

Comment:
we can init cache at creation time.

Reply:
What's the creation time? Could you please share which line of the code?